mne/viz/evoked.py

```diff
@@ -601,17 +601,17 @@ def _plot_image(data, ax, this_type, picks, cmap, unit, units, scalings, times,
     ax.set(ylabel=ylabel, xlabel='Time (%s)' % (time_unit,), title=t)
     _add_nave(ax, nave)
 
-    if show_names is not False:
-        if show_names == "all":
-            yticks = np.arange(len(picks)).astype(int)
-            yticklabels = np.array(ch_names)[picks]
-        else:
-            max_tick = len(picks)
-            yticks = [tick for tick in ax.get_yticks() if tick < max_tick]
-            yticks = np.array(yticks).astype(int)
-            # these should only ever be ints right?
+    if show_names == "all":
+        yticks = np.arange(len(picks)).astype(int)
```

Comment:
```suggestion
        yticks = np.arange(len(picks), dtype=int)
```

Reply:
The `dtype` of `arange` defaults to `int` if the inputs are `int` anyway
```
>>> np.arange(len([1])).dtype
dtype('int64')
```